tests/unit/other.tcl

```diff
@@ -368,3 +368,50 @@ start_server {tags {"other"}} {
     }
 }
 
+start_server {tags {"other"}} {
```

Comment:
we need `external:skip` due to AOFRW and DEBUG DIGEST.
I'd also suggest to move this to a file of it's own in the `integration` folder.
i regret not creating a specific file for defrag back then when i introduced it.

```suggestion
start_server {tags {"dismiss external:skip"}} {
```

also, please use gcov to improve this test till you see all the optimization `if`s we added are tested (e.g. add client output buffers, argv, etc)
big output buffers can be achieved by using a differing client that asks for something big, and doesn't read the response.
big argv array can be achieved by writing RESP directly to the socket and not providing the last argument.
e.g.
```tcl
    $rd write "*2\r\n$8192\r\n"
    $rd write $bigstr
```

Reply:
FYI (not sure if you know), you can do `make gconv` and then follow similar steps to the `lcov` target.

Reply:
I know, make sure to cover all new coed